blog/content/introtogonum.md

```diff
@@ -0,0 +1,145 @@
+
+## Gonum Philosophy
+The Gonum packages are designed to be simple, efficient and composable.
+There is a tension, especially in scientific computing, between ease and simplicity.
+Gonum, [like the Go language itself](https://commandcenter.blogspot.com/2012/06/less-is-exponentially-more.html),
+aims first to provide reliable and predictable code, while trying to maintain
+ease-of-use.
+Gonum is different from other popular scientific libraries, in that the Go language
+does not allow operator or method overloading.
+On the surface, this often means that the same algorithm requires a few more
+lines of code in Go.
+We fully recognize that sexy one-liners are rare with Gonum.
+
+We accept this cost because we believe it comes with an enormous benefit.
+It is easy to identify and explore the exact code segments being executed, and
+the Gonum function and method signatures make it clear what is happening.
+Science is built on understanding how an experiment was performed and understanding
+what the results are.
+We believe scientific code should be no different.
+In fact, Gonum is partly distinguished from other libraries in that the entire
+codebase can be run with pure Go -- you don't need to know another programming
+language to read Gonum code (that said, it is also easy to call into existing C libraries).
+
+This fundamental philosophy has many follow-on benefits.
+We find that one-liners often obscure the underlying operations, leading to
+less efficient code than a more targeted implementation.
+This is on top of an already fast language, Go approaches the speed of C.
+Go's language primitives are great for parallel computing, and in fact Gonum
+code can often be made to run in parallel by setting a flag.
+In other languages, it can hard to build and maintain large programs, especially
+for scientific coding where the final goals are not clear at the start of the project.
+Go was designed for ["programming in the large"](https://talks.golang.org/2012/splash.article),
+and a typed language combined with Go's package design and use of "interfaces"
+make it much more manageable to construct robust codebases and modify them for
+changing needs.
+This alone is enough to believe that Go is promising to build an ecosystem of
+scientific tools.
+To that you can add extremely legible compile errors, trivial package downloading,
+automatic imports, code formatting tools, easy code documentation, and the list goes on.
+
+Please check back for more discussion on the philosophy of Gonum and the design
+of specific libraries.
+
+## Getting started with Gonum
+### Downloading the libraries
+First, you need to [install Go](https://golang.org/doc/install) if it is not
+already installed on your machine.
+It is recommended that you use the most recent version of Go.
+Run 
+````
+go get -u -t gonum.org/v1/gonum````
+to download the main set of Gonum packages.
+Then run
+````go get -u -t gonum.org/v1/plot````
```

Comment:
did we actually say we'd break (yet another time) people using `github.com/gonum/plot`?

contrary to the other old `gonum/xxx` repos (e.g. `gonum/stat`) we can't leave `github.com/gonum/plot` intact *and* make it importable as `gonum.org/v1/gonum/plot` (except if we import `github.com/gonum/plot` inside `github.com/gonum/gonum`)